packages/client/src/views/SysAdmin/Config/Application/Tabs/BirthProperties/BirthDelayedFee.tsx

```diff
@@ -0,0 +1,194 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import * as React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import {
+  ApplyButton,
+  CancelButton,
+  Content,
+  Field,
+  HalfWidthInput,
+  InputContainer,
+  Label,
+  Value
+} from '@client/views/SysAdmin/Config/Application/Components'
+import { InputField } from '@opencrvs/components/lib/forms'
+import { IStoreState } from '@client/store'
+import {
+  FloatingNotification,
+  ListViewItemSimplified,
+  NOTIFICATION_TYPE,
+  ResponsiveModal
+} from '@opencrvs/components/lib/interface'
+import { BirthActionId } from '@client/views/SysAdmin/Config/Application'
+import { useIntl } from 'react-intl'
+import { messages } from '@client/i18n/messages/views/config'
+import { buttonMessages } from '@client/i18n/messages'
+import { getOfflineData } from '@client/offline/selectors'
+import {
+  callApplicationConfigMutation,
+  getCurrency,
+  getFormattedFee
+} from '@client/views/SysAdmin/Config/Application/utils'
+import { LinkButton } from '@opencrvs/components/lib/buttons'
+import { isString } from 'lodash'
+import { FormattedNumberCurrency } from '@opencrvs/components/lib/symbol'
+
+export function BirthDelayedFee() {
+  const intl = useIntl()
+  const dispatch = useDispatch()
+  const offlineCountryConfiguration = useSelector((store: IStoreState) =>
+    getOfflineData(store)
+  )
+  const [isValueUpdating, setIsValueUpdating] = React.useState(false)
+  const [showModal, setShowModal] = React.useState(false)
+  const toggleModal = () => setShowModal((prev) => !prev)
+  const [notificationStatus, setNotificationStatus] = React.useState<
+    'idle' | 'success' | 'error'
+  >('idle')
```

Comment:
This is probably used in a lot of places so why not create an Enum? And another improvement would be to add an `IN_PROGRESS` status and disable the actionButton when a mutation is `IN_PROGRESS`

Reply:
Done